includes/Modules/Analytics.php

```diff
@@ -292,6 +287,14 @@ protected function enqueue_gtag_js() {
 		 */
 		$gtag_opt = apply_filters( 'googlesitekit_gtag_opt', $gtag_opt );
 
+		if ( ! empty( $gtag_opt['linker'] ) ) {
+			wp_add_inline_script(
+				'google_gtagjs',
+				'gtag(\'set\', \'linker\', ' . wp_json_encode( $gtag_opt['linker'] ) . ' );'
+			);
+		}
```

Comment:
This inline script must be printed before the `gtag('js', new Date());` part (see https://developers.google.com/gtagjs/devguide/linker#basic_setup). Can you split out that respective part from line 260 and put it into its own inline script below here?

Reply:
Done!